src/constants/addresses.ts

```diff
@@ -22,6 +22,15 @@ const CELO_QUOTER_ADDRESSES = '0x82825d0554fA07f7FC52Ab63c961F330fdEFa8E8'
 const CELO_NONFUNGIBLE_POSITION_MANAGER_ADDRESSES = '0x3d79EdAaBC0EaB6F08ED885C05Fc0B014290D95A'
 const CELO_TICK_LENS_ADDRESSES = '0x5f115D9113F88e0a0Db1b5033D90D4a9690AcD3D'
 
+// bsc v3 addresses
+const BSC_V3_CORE_FACTORY_ADDRESSES = '0x128Ce3A3D48f27CE35A3F810cF2cddD2f6879b13'
+const BSC_ROUTER_ADDRESS = '0xa64Dd3A63fEed608660226112E748F93B87E650F'
```

Comment:
is this SwapRouter02? do we need this since we're on Universal Router? or just putting here cuz we haven't deprecated non-permit2 flow?

Reply:
@ilyamk